vhdbuilder/packer/convert-sig-to-classic-storage-account-blob.sh

```diff
@@ -86,9 +67,11 @@ echo "Converted $sig_resource_id to $disk_resource_id"
 # shellcheck disable=SC2102
 sas=$(az disk grant-access --ids $disk_resource_id --duration-in-seconds 3600 --query [accessSas] -o tsv)
 
-echo "Uploading $disk_resource_id to ${CLASSIC_BLOB}/${CAPTURED_SIG_VERSION}.vhd"
+echo "Logging into azcopy with MSI"
+azcopy-preview login --identity
```

Comment:
any reason we have to use an AZ Copy Preview binary ?

Reply:
might be an artifact of what's on the underlying dev container image, `azcopy` should have all the required functionality we need